README.md

```diff
@@ -4,20 +4,35 @@ This project is an operator (controller) that can be used to manage the installa
 
 ## Installation
 
-The **Istio Operator** has a dependency on the **Jaeger Operator** and **Kiali Operator**.  Before installing the Istio Operator, please make sure the
-Jaeger Operator and Kiali Operator have been installed.
+The **Istio Operator** has a dependency on the **Jaeger Operator**, **Elasticsearch Operator**  and **Kiali Operator**.  Before installing the Istio Operator, please make sure the
+Jaeger Operator, Elasticsearch Operator and Kiali Operator have been installed.
+
+### Installing the Elasticsearch Operator
+
+If available, the Elasticsearch operator (version 4.1) should be installed from the OperatorHub.
+
+Alternatively, to install the Elasticsearch operator manually, execute the following commands:
+
+```
+kubectl create ns openshift-logging # create the project for the elasticsearch operator
```

Comment:
@objectiser should be OpenShift command, However, unable to create new namespace starts with "openshift-"
Seems we have some restriction. https://bugzilla.redhat.com/show_bug.cgi?id=1428934
```
$ oc new-project openshift-logging
Error from server (Forbidden): project.project.openshift.io "openshift-logging" is forbidden: cannot request a project starting with "openshift-"
```

Version: OCP 4.1.3

Reply:
This is the reason for using the kubectl command instead. This is only relevant for dev/test scenarios, as normally the elasticsearch operator would be installed via operatorhub - so should be ok. I'll update the PR to add a comment.